app/javascript/lib/imperialWithoutInvestorCard.test.js

```diff
@@ -1,7 +1,20 @@
 import Action from './action';
+
+import Logger from '../src/Logger';
+
 import Imperial from './imperial';
+
+import BoardEurope from './board';
 import { Nation } from './constants';
 
+const ImperialFromLog = (log) => {
+  const game = new Imperial(BoardEurope, new Logger('unittests', 'stub-test-id'));
```

Comment:
And here.

Reply:
[76f61f0](https://github.com/Thrillberg/imperial/pull/729/commits/76f61f00f58e876e15be9e5e2d7e0ddfdbd8a432)